### ISSUE TITLE
Remove the public site from Hypha

### ISSUE THREAD

**Issue by frjo** [HyphaApp/hypha#3428]
The public site of Hypha is to be removed. It was a leftover from when this was a OTF project but no other organisations are using it.

There are some issue that need to be discussed.

Pages like the apply front page, the login page etc. are using the public site templates even do they are apply site pages. Mostly regarding header with nav links and the footer.

Footer is easy since the apply site pages have none. I think that works for the front page etc as well.

What to show in the header/nav?

Other issues we need to resolve?

- [x] https://github.com/HyphaApp/hypha/issues/3080
- [x] https://github.com/HyphaApp/hypha/pull/3432
- [x] Remove Google Translate Widget from Header Nav
- [x] https://github.com/HyphaApp/hypha/pull/3687
- [x] https://github.com/HyphaApp/hypha/pull/3689
- [x] #3694 
- [x] Update urlconf to only have single site and make apply site as default, remove all the code needed to separate out public and apply.

**Comment by frjo** [comment]
My suggestion for the header nav is that we only show the logo and a login button for anonymous users. If the user is logged in we show the same header/nav as on all other apple site pages.

**Comment by theskumar** [comment]
For header nav, since will be needed to back reference to main site. How about providing a configuration, similar to Hugo 

https://gohugo.io/content-management/menus/#define-in-site-configuration

```
SITE_MENU = {
      "main": [
         {
            "name": "Products",
            "pageRef": "/products",
            "weight": 10
         },
         {
            "name": "Hardware",
            "pageRef": "/products/hardware",
            "parent": "Products",
            "weight": 1
         },
         {
            "name": "Software",
            "pageRef": "/products/software",
            "parent": "Products",
            "weight": 2
         },
         {
            "name": "Services",
            "pageRef": "/services",
            "weight": 20
         },
         {
            "name": "Hugo",
            "params": {
               "rel": "external"
            },
            "pre": "\u003ci class=\"fa fa-heart\"\u003e\u003c/i\u003e",
            "url": "https://gohugo.io/",
            "weight": 30
         }
      ]
   }
}
```

**Comment by frjo** [comment]
Is a nav section like that needed? Not enough that the logo takes you to the public site?

I think of Hypha as an app, like Slack and Notion, not an extension of the organisations web site.

**Comment by theskumar** [comment]
While we discuss what and how to handle the nav. As per my discussion with @fourthletter it make sense to no have the current nav which is misleading. I'll create part1 of public part removal that removes the primary nav from fund listing/form and all the login/register/change password pages.

**Comment by theskumar** [comment]
As discussed internally, we want to the keep the listing page branded and use the public template while the application form, and login/register/password reset pages won't get the branding and navigation of the public site.

**Comment by wes-otf** [comment]
I started to take a look at the removal of the public site from the management scripts and it looks like the public and apply site get confused between `wagtailsiteupdate` and the `initialize` script. In the [`wagtailsiteupdate`](https://github.com/HyphaApp/hypha/blob/maintainence/remove-public-from-management-script/hypha/core/management/commands/wagtailsiteupdate.py#L16#L21) the Site object that is referenced for **apply is `id=2`** while **public is `id=3`**. While [`initialize`](https://github.com/HyphaApp/hypha/blob/maintainence/remove-public-from-management-script/hypha/core/management/commands/initialize.py#L49-#L57) references the **public site as `id=2`** and **apply as `id=3`**. 

Adding some debug logs to the migrations reveals that locally the **apply site is created with `id=3`** and the **public with `id=2`** thus making the `initialize` script the correct one for local operations. But, peering into the production database that's backward, with **apply having `id=2`** and **public having`id=3`**.

Hopefully this wasn't too hard to follow. This weirdness just might make the next migration a little strange as this flip-flopping needs to be considered. Will continue to work this.

**Comment by wes-otf** [comment]
Do we know if any adopters have the apply site set as default instead of public? Could that be used as an indicator rather than hard-coded ids for migrations.

**Comment by frjo** [comment]
Good catch @wes-otf! This will likely be one of the last parts we do for revoving public. The very last is cleaning up the db.

A good start would be to make the scripts above consistent.

We should also ask OTS and other implementors how it looks in their environments.

**Comment by wes-otf** [comment]
Sounds good to me! #3694 should make the scripts consistent & set the IDs to reflect the production instance for any new Hypha instances. 

When we're ready to start re-configuring the default site to apply and deleting the public, something like:

```
apply_site_qs = Site.objects.filter(root_page__slug="apply")
public_site_qs = Site.objects.filter(root_page__slug="public")
``` 
would allow for getting of the Site objects without having the rely on hard coded IDs if any of the adopters did deviate.

**Comment by theskumar** [comment]
Good to close this issue @frjo @wes-otf ?
